src/layers/Layer.js

```diff
@@ -6,47 +6,41 @@ import Observable, { unByKey } from 'ol/Observable';
  * an {@link https://openlayers.org/en/latest/apidoc/module-ol_layer_Layer-Layer.html ol/Layer}
  * @class
  * @param {Object} [options]
- * @param {string} [options.key] information about the key
- * @param {string} [options.name] The name of the new layer
- * @param {ol.layer} [options.olLayer] the {@link https://openlayers.org/en/latest/apidoc/module-ol_layer_Layer-Layer.html ol/Layer}
- * @param {radioGroup} [options.radioGroup] identifier to group layer in a group, toggle via a radio
- * @param {boolean} [options.isBaseLayer] if true this layer is the baseLayer
- * @param {Array<ol.layer>} [options.children] Layers
- * @param {boolean} [options.visible] If true layer is visible
- * @param {string} [options.copyright] Copyright-Statement
+ * @param {string} [options.key] Layer key.
```

Comment:
Please also add documentation about default values and required options.

Reply:
@friedjoff I set the name and olLayer properties as required, the rest are optional

Reply:
Please use the [JSDoc syntax for optional and default values](https://jsdoc.app/tags-param.html#optional-parameters-and-default-values).

Sorry, my comment from 4 days ago was a bit misleading. We don't need to specify which parameters are required, just the optional ones using the JSDoc syntax.